services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -1841,6 +1841,25 @@ module.exports = {
       return c;
     },
 
+    /**
+     *
+     */
+    contentCompanyField: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
+      const company = 'platform/content/company';
+      const { id, companyId } = input;
+      const content = await basedb.strictFindById('platform.Content', id, { projection: { type: 1 } });
+      const model = `platform/content/${dasherize(content.type)}`;
+      const body = new Base4RestPayload({ type: model });
+      if (companyId) body.setLink('company', { id: companyId, type: company });
```

Comment:
`company` should be nullable -- the `setLink` and `updateOne` should always happen regardless of a value. Update the definition for the input to allow the company to be nulled out and add a note about nullability? to the `companyId` field -- something like "If not present, the company relationship will be removed"